media-gfx/phototonic/patches/phototonic-2.1.patchset

```diff
@@ -0,0 +1,25 @@
+From 3cf9ca8c585474988f15827ce45eea5ad4672b30 Mon Sep 17 00:00:00 2001
+From: mazbrili <mazbrili@yahoo.com>
+Date: Tue, 30 Oct 2018 02:50:48 +0700
+Subject: change project file for haiku specific
+
+
+diff --git a/phototonic.pro b/phototonic.pro
+index 2468af5..5a2fa57 100644
+--- a/phototonic.pro
++++ b/phototonic.pro
+@@ -29,7 +29,11 @@ INCLUDEPATH += $$MINGWEXIVPATH/include
+ DEPENDPATH += $$MINGWEXIVPATH/include
+ 
+ PRE_TARGETDEPS += $$MINGWEXIVPATH/lib/libexiv2.a $$MINGWEXIVPATH/lib/libexpat.a $$MINGWEXIVPATH/lib/libz.a
+-}
++} 
```

Comment:
Oops, you added a white space here, in line 16, right after the `}`.
Could you please remove that whte space (not in directly in the patchset but in `phototonic.pro`) then amend the commit and regenerate the patchset?
(If you need help I can guide you.)
Thanks!

Reply:
ok .. fixed